File: Python/006_palindrome.py
# Task 7: Create a function that checks whether a given word is a palindrome or not.
def palidrome(word):
    count = 0
    for x in range(len(word)):
        # compares the x word with its reverse
        if word[x] == word[len(word)-1-x]:
            count += 1
    if count == len(word):
        p_word = "a palindrome."
    else:
        p_word = "not a palindrome."

    return p_word

if __name__ == "__main__":

    word = str(input("Enter a word(Check if its a palindrome): "))   
    p_word = palidrome(word)
    print(f"The word is {p_word}")

File: JavaScript/006_palindrome.js
const prompt = require("prompt-sync")({sigint:true});

function palindrome(word) {
    let count = 0;
    let p_word = "";
    for (let i = 0; i < word.length; i++) {
        console.log(word[i], word[word.length-1-i]);
        if (word[i] == word[word.length-1-i]) {
            count += 1;
        }
        
    }

    if (count == word.length) {
        p_word = "a palindrome.";
    } else {
        p_word = "not a palindrome.";
    }
    return p_word;
}

function main() {
    let word = String(prompt("Enter a word(Check if its a palindrome): "));
    let p_word = palindrome(word);
    console.log(`The word is ${p_word}`);
}

main()